controllers/operatorpolicy_controller.go

```diff
@@ -595,6 +644,28 @@ func (r *OperatorPolicyReconciler) applySubscriptionDefaults(
 		subSpec.CatalogSourceNamespace = catalogNamespace
 	}
 
+	if subscription.Namespace == "" {
```

Comment:
Do you think we could prefer the namespace of the existing subscription if the subscription already exists?

Reply:
Is it not in the relatedObjects on the status?

Reply:
Not if you create the policy after installing the operator.

Reply:
What I'm saying is that if the subscription already exists, use that namespace, otherwise look up the default.

This is to prevent switching namespaces in the event there is no suggested namespace but one is added later.